90_mender_boot_grub.cfg

```diff
@@ -16,10 +16,18 @@ else
     mender_root="${mender_kernel_root_base}${mender_boot_part}"
 fi
 
```

Comment:
Your signoff needs to match the author exactly. Looks like you have an extra middle "D" in the signoff string.